Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -16,25 +16,35 @@
     "version": {
         "Major": 3,
         "Minor": 2,
```

Comment:
Can you bump up minor version. I believe the new Linux support is not just a patch update.